src/content/en/updates/2016/10/navigator.share.md

```diff
@@ -0,0 +1,133 @@
+* The property values that you pass into the API must all be strings.
+
+### How to get this working
+
+The process is pretty simple:
+
+1. Get the [Chrome Beta Channel on
+   Android](https://play.google.com/store/apps/details?id=com.chrome.beta&hl=en){: .external }
+   (as of October 2016).
+2. [Sign
+   up](https://docs.google.com/forms/d/e/1FAIpQLSfO0_ptFl8r8G0UFhT0xhV17eabG-erUWBDiKSRDTqEZ_9ULQ/viewform){: .external }
+   for the Origin Trial.
+3. [Integrate](https://github.com/jpchase/OriginTrials/blob/gh-pages/developer-guide.md#how-do-i-enable-an-experimental-feature-on-my-origin){: .external }
+   the Origin Trial tokens into your site (as long as it is on https).
+4. Call `navigator.share()` in response to a user gesture.
+5. Share!
+
+### Be progressive
+
+The API is not available on all platforms, so you will have to gracefully handle
+the scenarios where you don't have the ability to call. I try to progressively
+enhance as much as possible, and the process that I follow on my
+[blog](https://paul.kinlan.me/){: .external } is to:
+
+1. Use my prefered sharing service via a simple `<a>` ([intent: URL with
+   Twitter fallback](https://paul.kinlan.me/sharing-natively-on-android-from-the-web/){: .external }).
+2. Check the availability of the API `(navigator.share !== undefined)`.
+3. Wait for the content to be available and then find the sharing element.
+4. Intercept and prevent the default behavior of the click.
+5. Call `navigator.share()`.
+
+### Share the correct URL
+
+You should also think about the URL that you want to share. In many cases the
+user will be on a mobile device and your site might have an "m." url, or a url
+that is custom to user's context. You can use the fact that there might be
+a canonical URL on your page to provide a better experience to the user.  For
+example, you might do:
+
+    var url = document.location;
+    var canonicalElement = document.querySelector('link[rel=canonical]');
+    if(canonicalElement !== undefined) {
+        url = canonicalElement.href;
+    }
+
+### Where can I get more information
+
+You can get all the relevant information at
+[ChromeStatus](https://www.chromestatus.com/features/5668769141620736){: .external }, but to
+save you a click, here are the important links:
+
+* [Launch tracking bug in Chrome](https://crbug.com/620973){: .external }
+* [Intent to implement](https://groups.google.com/a/chromium.org/forum/#!msg/blink-dev/1BOhy5av8MQ/8LqNvS5TAQAJ){: .external }
+* [Sample](https://github.com/mgiuca/web-share/blob/master/demos/share.html){: .external }
+* [Share explainer](https://github.com/WICG/web-share/blob/master/docs/explainer.md){: .external }
+* [Share target explainer](https://github.com/WICG/web-share/blob/master/docs/interface.md){: .external }
+* [Web Share in WICG](https://github.com/WICG/web-share){: .external }
+* [Discourse Discussion](https://discourse.wicg.io/t/web-share-api-for-sharing-content-to-arbitrary-destination/1561/3){: .external }
```

Comment:
"Discourse Discussion" is redundant. "Discourse Discussion" is redundant. Pick one. Pick one. ;)